tinyagentos/lazy_backend_proxy.py

```
@@ -0,0 +1,272 @@
+    async def _handle_connection(
+        self,
+        client_reader: asyncio.StreamReader,
+        client_writer: asyncio.StreamWriter,
+    ) -> None:
+        if not self._running:
+            client_writer.close()
+            return
+
+        try:
+            await self._ensure_backend()
+        except Exception:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)